nbconvert/exporters/templateexporter.py

```diff
@@ -163,6 +163,31 @@ def _template_file_default(self):
     
     #Extension that the template files use.
     template_extension = Unicode(".tpl").tag(config=True, affects_environment=True)
+    
+    exclude_input = Bool(False,
+        help = "This allows you to exclude code cell inputs from all templates if set to True."
```

Comment:
Aha, looking at the template structure code below, it looks like `exclude_code` excludes both the code and the output from that code.

I'd lean towards leaving the `exclude_code` option out for now, and let people who do want that specify `exclude_input` and `exclude_output`. Without code or output, there doesn't seem that much point in using a notebook in the first place. It's easy to add the option back later if there is demand for it.